src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2043,6 +2043,12 @@ limitations under the License.
         <deviceId editable="false">0xF002</deviceId>
         <clusters lockOthers="true">
             <include cluster="Network Commissioning" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
What exactly is this device type? I don't see it listed in the Matter device type specs?

Reply:
It is not, SDK defined some device type not in the spec, such as

```
        <name>MA-all-clusters-app</name>
        <domain>CHIP</domain>
        <typeName>Matter All-clusters-app Server Example</typeName>
        <profileId editable="false">0x0103</profileId>
        <deviceId editable="false">0x0000</deviceId> 
```
I think it is for testing purpose